### PR TITLE
feat: 176569436: domain based follow logic for links in description

### PR DESCRIPTION
**Pull request by kuhdm0**
https://www.pivotaltracker.com/n/projects/2457572/stories/176569436

### CODE REVIEW

File: packages/mwp-core/src/util/linkify.js [hunk 1]
```diff
@@ -5,6 +5,24 @@ var urlRegex = new RegExp(
 	/(?:(?:(?:[a-z]+:)?\/\/)|www\.)(?:\S+(?::\S*)?@)?(?:localhost|(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])(?:\.(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])){3}|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,})))(?::\d{2,5})?(?:[/?#]([^\s"()]|[(][^\s"]*?[)])*)?/gi
 );
 
+const createRelTag = (isDomainLogicEnabled, domains, link, target) => {
```
Comment (andeliv) [inline packages/mwp-core/src/util/linkify.js:8]: `rel` is an attribute, not a tag so I'd suggest to rename this func to getRelAttr

File: packages/mwp-core/src/util/linkify.js [hunk 2]
```diff
@@ -31,9 +53,14 @@ const createLink = (options: Object) => (href: string): string => {
  * @param {Object} options optional modifiers
  * @return {String} The modified text.
  */
-export default function linkify(text: string, options?: Object = {}): string {
+export default function linkify(
+	text: string,
+	options?: Object = {},
+	isDomainLogicEnabled?: boolean,
```
Comment (andeliv) [inline packages/mwp-core/src/util/linkify.js:59]: Do we need this separate flag? How about just validating that `domains` is defined?

File: packages/mwp-core/src/util/linkify.js [hunk 3]
```diff
@@ -31,9 +53,14 @@ const createLink = (options: Object) => (href: string): string => {
...
+	domains: Array<string>
```
Comment (andeliv) [inline packages/mwp-core/src/util/linkify.js:60]: I'd rename that to smth more specific like `followedExternalDomains`

**Review by UnknownNPC — APPROVED** [review]
LGTM

**Review by andeliv — APPROVED** [review]

**Pull request merged**